pkgs/development/libraries/sregex/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "sregex";
+  version = "0.0.1";
+
+  src = fetchFromGitHub {
+    owner = "openresty";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-HZ9O/3BQHHrTVLLlU0o1fLHxyRSesBhreT3IdGHnNsg=";
+  };
+
+  makeFlags = [ "PREFIX=$(out)" "CC=${stdenv.cc}/bin/cc" ];
```

```suggestion
  makeFlags = [ "PREFIX=$(out)" "CC:=$(CC)" ];
```
As "${stdenv.cc}/bin/cc" is not always correct, for example cross compilers are prefixed with the target platform.

---

I fixed, thanks